internal/pkg/kernel/kernel.go

```diff
@@ -122,21 +122,25 @@ func Build(kernelVersion string, kernelName string, root string) (string, error)
 		return "", fmt.Errorf("failed to create version dest: %s", err)
 	}
 
-	if !util.IsFile(kernelImage) {
-		if !util.IsFile(kernelImage + ".gz") {
-			return "", errors.New("Could not locate kernel image")
-		} else {
-			kernelImage = kernelImage + ".gz"
-		}
+	if util.IsFile(path.Join(root, "/boot/vmlinuz-"+kernelVersion)) {
```

Comment:
@gmkurtzer this would read cleaner and be more extensible to additional locations if the locations were placed in a slice of strings, then use the "for ... range" to iterate locations until found.    also, can use the variable "kernelDrivers" instead of retyping the path...

Reply:
I got the slice integrated, but I don't get your second point about `kernelDrivers` and retyping the path.

Thanks!